services/frontend-v3/src/i18n/en_CA.json

```
@@ -157,6 +157,9 @@
   "profile.branch": "Branch",
   "profile.career.content.name": "Description",
   "profile.career.header.name": "Job title",
+  "profile.career.header.tooltip": "It is recommended to use the information synced from GEDS. However if you edit your job title, in order to display both a customized French and English title, you must change the app language and save each title in the form accordingly.",
```

GEDS has been renamed to GCdirectory
```suggestion
  "profile.career.header.tooltip": "It is recommended to use the information synced from GCdirectory. However if you edit your job title, in order to display both a customized French and English title, you must change the app language and save each title in the form accordingly.",
```